### PR TITLE
Email plot bug fix and enhancement

### PR DESCRIPTION
**Pull request by rhKashif**
- Removed duplicate reviews bug in tables
- Increases minimum number of reviews for display in `plot_trending_games_sentiment_table` to 10

### CODE REVIEW

File: report/lambda_function.py [hunk 1]
```diff
@@ -633,9 +670,9 @@ def handler(event, context) -> None:
         game_df = format_database_columns(game_df)
 
         create_report(game_df, config["DASHBOARD_URL"])
-        print("Report created.")
+        # print("Report created.")
```
Comment (DMcCallion) [inline report/lambda_function.py]: Commented out code 👀

File: report/lambda_function.py [hunk 2]
```diff
@@ -633,9 +670,9 @@ def handler(event, context) -> None:
...
 
-        email_subscribers(conn, config)
+        # email_subscribers(conn, config)
```
Comment (DMcCallion) [inline report/lambda_function.py]: And here uh oh

**Review by DMcCallion — APPROVED** [review]
Nice fixes!

File: report/lambda_function.py (hunk 2 above)
Comment (angelikavilde) [inline report/lambda_function.py]: do we need those comments?

Comment (angelikavilde) [reply report/lambda_function.py]: can we add `except`?

File: report/lambda_function.py [hunk 3]
```diff
@@ -208,6 +208,28 @@ def get_most_reviewed_release(df_releases: DataFrame) -> str:
 
     return df_ratings.head(1)["title"][0]
 
+  
+def calculate_sum_sentiment(sentiment: float, score: int) -> float:
+      """
```
Comment (gawsalya) [inline report/lambda_function.py]: remove space here

File: report/lambda_function.py (hunk 2 above)
Comment (rhKashif) [reply report/lambda_function.py]: Hmm, this is an interesting proposition but since it is mostly Dan's code, I am going to leave it as it is for now.

**Pull request merged**